src/openrct2/object/ObjectManager.cpp

```diff
@@ -232,6 +233,21 @@ class ObjectManager final : public IObjectManager
         return objects;
     }
 
+    static rct_string_id GetObjectSourceGameString(const rct_object_entry * entry)
+    {
+        uint8 source = (entry->flags & 0xF0) >> 4;
+        switch (source) {
+            case OBJECT_SOURCE_RCT2:
```

Comment:
The inside of this switch block should be intended one "tab" fewer. (Since `case` and `default` are labels.)